Specs/Scene/CameraSpec.js

```diff
@@ -3148,6 +3148,17 @@ describe("Scene/Camera", function () {
     }).toThrowDeveloperError();
   });
 
+  it("get pick ray returns undefined if the Scene is not fully rendered", function () {
+    const windowCoord = new Cartesian2(
+      scene.canvas.clientWidth / 2,
+      scene.canvas.clientHeight
+    );
+
+    scene.canvas.clientWidth = 0;
+    const ray = camera.getPickRay(windowCoord);
+    expect(ray).toEqual(undefined);
```

Comment:
```suggestion
    expect(ray).toBeUndefined();
```

This is knit-picky, but there's a specific [matcher](https://jasmine.github.io/api/2.7/matchers.html) for this case, you don't always need to use `toEqual`.